### PR TITLE
Add proper symbol filtering to Imports.AddLookupSymbolsInfoInUsings.

### LINKED ISSUE #7101: CodeCompletion offers non-static method via "using static Class;"
image

Program.cs:
```csharp
using static ClassWithNonStaticMethod;
class Program
{
    static void Main(string[] args)
    {
        var instace = new Program();
        instace.NonStaticMethod();
    }

    private void NonStaticMethod()
    {
        MathMin(0, 1);
    }
}
```

ClassWithNonStaticMethod.cs:
```csharp
using System;

class ClassWithNonStaticMethod
{
    public static int MathMax(int a, int b)
    {
        return Math.Max(a, b);
    }

    public int MathMin(int a, int b)
    {
        return Math.Min(a, b);
    }
}
```

Fun fact: Bug was discovered while using Xamarin Studio :)

### PR DESCRIPTION
**Pull request by AlekseyTs**
Fixes #7101.

@dotnet/roslyn-compiler Please review.

### CODE REVIEW

**Comment by cston** [comment]
LGTM

**Comment by AlekseyTs** [comment]
@dotnet/roslyn-compiler Please review, need a second sign-off.

File: src/Compilers/CSharp/Portable/Binder/Imports.cs [hunk 1]
```diff
@@ -716,6 +691,41 @@ internal bool IsUsingAlias(string name, bool callerIsSemanticModel)
             }
         }
 
+        private static bool IsValidLookupCandidate(Symbol symbol)
```
Comment (jaredpar) [inline src/Compilers/CSharp/Portable/Binder/Imports.cs]: IsValidLookupCandidateInUsings?

Comment (AlekseyTs) [inline src/Compilers/CSharp/Portable/Binder/Imports.cs]: Sure, I'll rename the method.

**Comment by VSadov** [comment]
LGTM

**Comment by jaredpar** [comment]
LGTM

**Pull request merged**